src/domain/open311Backend/types.ts

```diff
@@ -0,0 +1,85 @@
+/* eslint-disable camelcase */
+
+export type Open311BackendConfig = {
+  apiKey: string;
+  url: string;
+  serviceCode: string;
+};
+
+export type Open311PostServiceRequestCamelCase = {
+  description: string;
+  title: string | null; // CitySDK specific
```

Comment:
Great to see which fields are CitySDK specific (I didn't know what that was, but I see it is mentioned in https://dev.hel.fi/apis/open311